pkg/virt-controller/watch/instancetype/upgrade.go

```diff
@@ -236,9 +241,21 @@ func (c *UpgradeController) updateWithSuccess(crUpgrade *instancetypev1beta1.Con
 	return nil
 }
 
-func (c *UpgradeController) updateWithFailure(crUpgrade *instancetypev1beta1.ControllerRevisionUpgrade) error {
+func (c *UpgradeController) updateWithFailure(crUpgrade *instancetypev1beta1.ControllerRevisionUpgrade, updateErr error) error {
```

Comment:
Can you pass `string` parameter instead of `error`? The error is only used as message passed to the condition.